iiasa/ixmp
# Add `.utils.sphinx-linkcode-github`

Based on [this code example](https://github.com/sphinx-doc/sphinx/issues/1556#issuecomment-1010273175), this extends `sphinx.ext.autocode` and `sphinx.ext.linkcode` to link to file source on GitHub.

This will obviate the need to use `sphinx.ext.viewcode` on ixmp stack repositories.

## How to review

- View the built docs for this branch.
- Try some of the [source] links.
- Ensure these work.
- View the docs for the new module, ensure they are complete.
- Note the CI checks all pass.

## PR checklist
